### ISSUE TITLE
Mute and Volume controls should be shown when explicitly listed as controls

### ISSUE THREAD

**Issue by bseib** [sampotts/plyr#1603]
### Expected behaviour

The Mute and Volume controls should be shown when a specific list of controls is called out, i.e. `controls: ['play', 'progress', 'current-time', 'mute', 'volume']`.

### Actual behaviour

When a list of specific controls is passed in, the Mute and Volume controls are not shown.

![image](https://user-images.githubusercontent.com/1313561/67499727-8ffbbd80-f64f-11e9-8cff-fbaf1f688713.png)

### Steps to reproduce

```
const player = new Plyr('#player', {
      controls: ['play', 'progress', 'current-time', 'mute', 'volume'],
  });
```

https://codepen.io/bseib/pen/wvvddBV

### Environment

- Browser: Chrome
- Version: 77.0.3865.120 (Official Build) (64-bit)
- Operating System: Windows
- Version: 10

### Link to where the bug is happening

It looks like the `hidden` attribute is being set unexpectedly on the Mute and Volume controls:

![image](https://user-images.githubusercontent.com/1313561/67500364-8b83d480-f650-11e9-9143-d82b5bff84e4.png)

**Comment by jetlej** [comment]
I'm seeing this too

**Comment by Nickmob** [comment]
The same with Ubuntu, Windows 10, Chrome 78 and Chromium 78.
To fix this I have to enable in about://flags: Experimental Web Platform features - then enables, controls show as expected.

**Comment by ChubbyDuck** [comment]
This is caused by this handler https://github.com/sampotts/plyr/blob/74e3990604698b1f2af3ed51e65b0ca820c1d624/src/js/listeners.js#L382
It seems that by the time of that event, for some reason there are no audio tracks in chrome(in firefox everything is okay), or, at least, plyr.hasAudio method gives such output. I'm not sure how to fix that properly, but this handler can be just deleted as a temp solution.

**Comment by shravan2x** [comment]
I've been facing this issue for at least 6 months. The volume slider doesn't show up until the user clicks the seekbar.

**Comment by shravan2x** [comment]
I've been facing this issue for at least 6 months. The volume slider doesn't show up until the user clicks the seekbar.

@ChubbyDuck Could you provide a working example?

**Comment by bpeab** [comment]
Facing the same issue.

**Comment by dalejandre** [comment]
This issue is solved overwritting this class:

`platform-android media-player .plyr [hidden] {
display: flex !important;
}`

Modifying this classes should solve issue for other platforms
`.plyr .plyr__volume[hidden] {
display: flex !important;
}
.plyr .plyr__volume button[hidden]{
display: flex !important;
}`
Its not the prettiest solution, the problem is the hidden property of DOM element. It should not be appear in elements. At least, this solution works for us.

**Comment by sampotts** [comment]
This should be fixed in 3.5.7 👍
